arch-update/arch-update

```diff
@@ -112,8 +112,13 @@ if args.aur:
 
 update_count = len(updates)
 if update_count > 0:
-    info = str(update_count) + ' updates available'
-    short_info = str(update_count) + ' updates'
+    if update_count == 1:
```

Comment:
instead of
```
if update_count > 0:
    if update_count == 1:
    else:
```

you should better use
```
if update_count == 1:
elif update_count > 1:
```

Reply:
Please see the whole segment:
```python3
update_count = len(updates)
if update_count > 0:
    if update_count == 1:
      info = str(update_count) + ' update available'
      short_info = str(update_count) + ' update'
    else:
      info = str(update_count) + ' updates available'
      short_info = str(update_count) + ' updates'

    matches = matching_updates(updates, args.watch)
    if matches:
        info += ' [{0}]'.format(', '.join(matches))
        short_info += '*'
    print(message.format(label, args.updates_available_color, info))
    print(message.format(label, args.updates_available_color, short_info))
elif not args.quiet:
    print(message.format(label, args.base_color, 'system up to date'))
```

If I do it as you propose the code would be less DRY:

```python3
update_count = len(updates)
if update_count == 1:
      info = str(update_count) + ' update available'
      short_info = str(update_count) + ' update'
      matches = matching_updates(updates, args.watch)
      if matches:
          info += ' [{0}]'.format(', '.join(matches))
          short_info += '*'
      print(message.format(label, args.updates_available_color, info))
      print(message.format(label, args.updates_available_color, short_info))
elif update_count > 1:
      info = str(update_count) + ' updates available'
      short_info = str(update_count) + ' updates'
      matches = matching_updates(updates, args.watch)
      if matches:
          info += ' [{0}]'.format(', '.join(matches))
          short_info += '*'
      print(message.format(label, args.updates_available_color, info))
      print(message.format(label, args.updates_available_color, short_info))
elif not args.quiet:
    print(message.format(label, args.base_color, 'system up to date'))
```

Reply:
Sorry, I missed that! In this case you are right, of course.